.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: Run watir-rails CI
+
+'on':
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
Is this Docker image name? What about using some more light-weight distro like Alpine in the hopes of everything being faster and less resource-hungry? It's more like nice-to-have request and could do later too.

Reply:
It's GH runner type ([here](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on)), we can use only ubuntu as a linux one.

Reply:
Funny that `ubuntu-latest` refers to Ubuntu 18, but there's also `ubuntu-20`. Maybe it's better to configure a specific version (either 18 or 20) in case GitHub decides that latest means something else and then our build will stop working due to version differences?

Reply:
It's actually happening right now https://github.com/actions/virtual-environments/issues/1816 and I tested in on 20.04 explicitly, so I'm counting that as `either 18.04 or 20.04, but will be only 20.04 soon`.

It will be updated from time to time by github, but 
1. I doubt that something will break
2. It should be fixed anyway
3. I'm planing to add explicit versions in https://github.com/watir/watir-rails/pull/30/commits/f77638e9b33315eafd22f78b728b251cc82b3ed8, so this one is just `test on any ubuntu, preferably latest one`.

Reply:
It's always good to specify an exact version instead of using `latest` - then it is also easier to troubleshoot/test things out on a VM/Docker using the same version if a situation arises. Breaking changes might occur when GitHub switches latest to something different and then it's hard to troubleshoot again since it might take some time to understand that this change was done in the first place. If we fixate on a specific version and upgrade manually and things break then it's pretty easy to conclude that there are some differences between Linux distros.

Also, using word `latest` also causes the aforementioned problem where latest is not actually latest - I already went for that since I assumed (for obvirous reasons, right?) that latest means Ubuntu 20.

Reply:
Switched to ubuntu-20.04.